src/textual/app.py

```diff
@@ -1608,9 +1608,8 @@ async def invoke_ready_callback() -> None:
                                 with redirect_stdout(redirector):  # type: ignore
                                     await run_process_messages()
                         else:
-                            null_file = _NullFile()
-                            with redirect_stderr(null_file):
-                                with redirect_stdout(null_file):
+                            with redirect_stderr(None):
```

Comment:
I was surprised by this! I can't find it documented that redirecting to `None` works.

Reply:
Now that you mention it, me neither...
The type hints gave it away. It was complaining about the argument type and it said that "Type _NullFile" cannot be assigned to type "IO[str] | None", from which I inferred `None` would redirect to the void.